File: check_db.js
const { Client } = require('pg');
const client = new Client({
  user: 'postgres',
  host: 'localhost',
  database: 'civicpulse_db',
  password: 'civic@12',
  port: 5432,
});
client.connect();
client.query('SELECT application_number, department FROM service_applications', (err, res) => {
  console.log(err ? err.stack : res.rows);
  client.end();
});

File: check_db.py
import psycopg2
try:
    conn = psycopg2.connect(dbname="civicpulse_db", user="postgres", password="civic@12", host="localhost", port="5432")
    cur = conn.cursor()
    cur.execute("SELECT application_number, department FROM service_applications;")
    rows = cur.fetchall()
    print("TOTAL ROWS:", len(rows))
    for row in rows:
        print(row)
    cur.close()
    conn.close()
except Exception as e:
    print(e)
